test/init/auto/init-auto.test.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable node/no-unpublished-require */
+'use strict';
+
+const { run } = require('../../utils/test-utils');
+const firstPrompt = 'Will your application have multiple bundles?';
+const fs = require('fs');
+const { join } = require('path');
+
+describe('init auto flag', () => {
+    it('should prompt with w/o auto flag', () => {
+        const { stdout, stderr } = run(__dirname, ['init'], false);
+        expect(stdout).toBeTruthy();
+        expect(stderr).toBeFalsy();
+        expect(stdout).toContain(firstPrompt);
```

Comment:
Can we improve it, and answer on all questions?

Reply:
@evilebottnawi the future questions changes when we answer with different values. So should should I test all possible question flows or one of them?

Reply:
Ideally - yes

Reply:
This would take a lot of CI time and memory.
This may result in being flaky, as if we would add/change/remove one question from generator, Or we change the functionality used by generator, it would break the while test suite for init.
In past, tests for init were removed because of this reason
Rest I would do as you suggest :)

Reply:
I guess one full example would be good, we can check to the point it generates the config.

Reply:
> @evilebottnawi This would take a lot of CI time and memory.

> This may also result in tests being flaky, as if we would add/change/remove one question from generator, Or we change the external functionality which is some where used by generator, it would break the whole test suite for init.

> In past, these type of tests for init were removed because of this reason

> Rest I would do as you suggest :)

When something removed we need remove it from tests, this is testing, which allows us to understand that everything is fine, the tests are stable, and if something doesn’t work, then we have an error, I don’t know why someone decided to delete them, this is a very, very bad idea

Reply:
> I guess one full example would be good, we can check to the point it generates the config.

Yes, as minimum we should have test for it

Reply:
Great, thanks for suggestions.